src/webgpu/shader/validation/expression/call/builtin/subgroupBroadcastFirst.spec.ts

```diff
@@ -0,0 +1,172 @@
+export const description = `
```

Comment:
coder's choice, but could be folded in with the subgroupBroadcast test file?

Reply:
Thought about that, but they take a different number of arguments.